source/simulator/core.cc

```diff
@@ -130,7 +130,9 @@ namespace aspect
             this_mpi_process(mpi_communicator)
             == 0)),
 
-    computing_timer (pcout, TimerOutput::never,
+    computing_timer (mpi_communicator,
+                     pcout,
```

Comment:
Ah, I see. This change then still strikes me as the correct thing to do. Would you agree, @tjhei ?